header_compilation_tests.sh

```diff
@@ -38,13 +38,18 @@ out_uptodate()
     done
 }
 
-HCC=/opt/rocm/hcc/bin/hcc
+rocm_path=/opt/rocm
```

Comment:
Why are we hard coding here? What if we install to /usr/local instead? Why are we assuming /opt/rocm?

Reply:
That is a default, you override with ROCM_PATH.   Not my PR but that is fine by me.  Same is done in install.sh